app/(afterLogin)/(payroll)/payroll/page.tsx

```
@@ -77,6 +78,14 @@ const Payroll = () => {
   const { data: allEmployees } = useGetAllUsersData();
   const { data: employeeData } = useGetAllUsers();
   const [searchValue, setSearchValue] = useState<{ [key: string]: string }>({});
+
+  // Add the export hook for fetching all payroll data without pagination
+  const {
+    data: allPayrollData,
```

ESLint

[error] 'allPayrollData' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)